### ISSUE TITLE
Most tours broken currently due to UI changes

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#2472]
I found this issue while searching for [tours I could use in Selenium tests](https://github.com/galaxyproject/galaxy/issues/11413).

The TL;DR version is that all of the tours are broken; 30 of the 33 tours use a CSS .class selector (.fa.fa-upload) that is no longer used on the upload button.  The other three tours failed waiting for #btn-reset to become clickable; note that most of the other 30 would have failed for this reason as well if they hadn't already failed trying to find the upload button.

Having said that, those two problems can likely be fixed with find/grep/awk scripts and I would like to investigate how much effort it would be to get the tours working again.  I am willing to bet that it is less effort than crafting an entire new set of Selenium tests from scratch and the tours provide some nice real world use cases to test against.  Once the tours are part of the selenium test regime it should be much easier to keep them updated as we will know as soon as one of them breaks.

_Originally posted by @ksuderman in https://github.com/galaxyproject/training-material/issues/840#issuecomment-811322997_

**Comment by hexylena** [comment]
@ksuderman moved this into it's own issue. thanks for adding this bug report! We really need more stable UI identifiers for these buttons :(
